Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb_example.py

```diff
@@ -106,3 +118,10 @@ def meas():
 
 res.plot_fidelity()
 plt.show()
+
+# verify/save the random sequences created during the experiment
+rb.save_sequences_to_file("sequences.txt")  # saves the gates used in each random sequence
+# rb.save_command_mapping_to_file('commands.txt')  # saves mapping from "command id" to sequence
+# rb.print_sequences()
+# rb.print_command_mapping()
+# rb.verify_random_sequences() # simulates random sequences to ensure they recover to ground state. takes a while...
```

Comment:
The command is `rb.verify_sequences()`, no?

Reply:
You're right. Fixed it